packages/css-framework/src/components/_container.scss

```diff
@@ -1,39 +1,25 @@
 .rn-container {
   max-width: $navy-content-width;
-  margin: 0 auto;
-  padding: spacing(3);
-
-  &.rn-container--flush {
-    padding: 0;
-  }
-
-  &.rn-container--flush-horizontal {
-    padding-left: 0;
-    padding-right: 0;
-  }
-
-  &.rn-container--flush-vertical {
-    padding-bottom: 0;
-    padding-top: 0;
-  }
+  margin-left: auto;
+  margin-right: auto;
+  padding-left: spacing(3);
+  padding-right: spacing(3);
 
   &.rn-container--small {
-    padding: spacing(2);
+    padding-left: spacing(2);
+    padding-right: spacing(2);
   }
 
   &.rn-container--large {
-    padding: spacing(5);
+    padding-left: spacing(5);
+    padding-right: spacing(5);
   }
 }
 
 .rn-pagecontainer {
```

Comment:
what's the difference between `.rn-container` and `.rn-pagecontainer`? Can we just use `.rn-container`?

Reply:
well spotted